### ISSUE TITLE
Git commit fails using neovim

### ISSUE THREAD

**Issue by ggeorgakoudis** [rhysd/vim-grammarous#44]
Git commit fails when vim-grammarous is loaded, it's fine otherwise. The version of neovim is v0.2.0 and I'm running on a mac: Darwin mac 16.6.0 Darwin Kernel Version 16.6.0: Fri Apr 14 16:21:16 PDT 2017; root:xnu-3789.60.24~6/RELEASE_X86_64 x86_64.

The error is:
```
error: There was a problem with the editor 'vi'.
Please supply the message using either -m or -F option.
```

Doing ```echo $?``` immediately after exiting vim shows a return value of 1

**Comment by rhysd** [comment]
Thank you for reporting this. I haven't seen this ever.

> Git commit fails

Could you please describe more defails what 'fails' means? What is a timing of the error? Where the error message was output? (to stdout? or message area in nvim?) And please tell me your config of vim-grammarous. I need to reproduce to fix the error.

**Comment by ggeorgakoudis** [comment]
When you do `git commit` in a repo and without providing a message using -m, the editor of choice opens up to write your commit message. My editor is neovim and I'm using the vim-grammarous plugin, which is great by the way. After I write my commit message and save and quit in vim (:wq), then this error message appears on the command line. Doing an `echo $?` shows a return value of 1, which signifies an error. Committing of course fails.

**Comment by rhysd** [comment]
Thank you for details. Do you write any config for vim-grammarous in your init.vim?

**Comment by ggeorgakoudis** [comment]
No, I don't have any configuration for vim-grammarous in the init.vim
